backup-parameter-store/src/main/scala/com/gu/aws/wrappers.scala

```diff
@@ -0,0 +1,81 @@
+package com.gu.aws
+
+import java.io.ByteArrayInputStream
+import com.amazonaws.auth.{ AWSCredentialsProviderChain, InstanceProfileCredentialsProvider }
+import com.amazonaws.auth.profile.ProfileCredentialsProvider
+import com.amazonaws.services.s3.AmazonS3ClientBuilder
+import com.amazonaws.services.s3.model.{ ObjectMetadata, PutObjectRequest, PutObjectResult, SSEAwsKeyManagementParams }
+import com.amazonaws.services.simplesystemsmanagement.{ AWSSimpleSystemsManagement, AWSSimpleSystemsManagementClientBuilder }
+import com.amazonaws.services.simplesystemsmanagement.model.GetParametersByPathRequest
+
+import scala.collection.JavaConverters._
+import scala.annotation.tailrec
+
+object AwsConfig {
+  val provider = new AWSCredentialsProviderChain(
+    new ProfileCredentialsProvider("frontend"),
+    InstanceProfileCredentialsProvider.getInstance())
+  val region = "eu-west-1"
+  val kmsKeyAlias: String = "arn:aws:kms:eu-west-1:642631414762:alias/FrontendConfigKey"
+}
+
+class ParameterStore {
+
+  private lazy val client: AWSSimpleSystemsManagement = AWSSimpleSystemsManagementClientBuilder
```

Comment:
Would be good to inject these client dependencies (in all the aws wrappers) either as constructor args or even better just passing as an argument to each method.

Reply:
In this scenario I don't see an advantage in doing this

Reply:
Is that because you think the app is too small to have any benefits?

Reply:
In backup-parameter-store, the AWSSimpleSystemsManagement and S3 clients aren't used anywhere else. 

The only reason I can see for injecting them would be for unit testing. I decided against unit testing my client wrappers.